### PR TITLE
DEL-317 Monitoring AMQ addressspace/queue creation/deletion during upgrade

### PR DESCRIPTION
**Pull request by b1zzu**
Approach:
* Create an AddressSpace
* Create an Address
* Delete the Address (even the previous steps failed)
* Delete the AddressSpace (even the previous steps failed)
* If just one of the above steps fails report the error in the metric `amq_crud` and set as downtime the total of performing the above steps
* repeat without wait

Todo:
- Update grafana dashboard
- Create ServiceAccount

### CODE REVIEW

**Review by wei-lee — APPROVED** [review]
/lgtm

**Comment by STEPHINRACHEL** [comment]
Deployed the app with dashboard enabled and AMQ CRUD check has started,
but getting no data points in the graph, Is it expected?
Screenshot 2020-07-03 at 09 37 32

**Comment by b1zzu** [comment]
@STEPHINRACHEL it could take ~5m before the first cycle is completed, can you try again to see if there is something?

**Comment by STEPHINRACHEL** [comment]
verified locally
/lgtm

File: main.go [hunk 1]
```diff
@@ -49,7 +50,7 @@ func init() {
 }
 
 func handler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "Hello World!")
+	fmt.Fprintf(w, "Fuck You!")
```
Comment (wei-lee) [inline main.go:53]: lol. personally i have no problem with this, but given that this is an open source project, we might want be more careful about the languages 😉

Comment (b1zzu) [reply main.go:53]: It was just a matter of time before committing one :see_no_evil:

Comment (b1zzu) [reply main.go:53]: https://github.com/integr8ly/workload-web-app/pull/19

**Pull request merged**